custom_components/adjustable_bed/const.py

```
@@ -167,13 +167,27 @@
+OCTO_STAR2_SERVICE_UUID: Final = "0000aa5c-0000-1000-8000-00805f9b34fb"
+OCTO_STAR2_CHAR_UUID: Final = "00005a55-0000-1000-8000-00805f9b34fb"
+
 # Octo PIN keep-alive interval (seconds)
 # Octo beds drop BLE connection after ~30s without PIN re-authentication
 OCTO_PIN_KEEPALIVE_INTERVAL: Final = 25
 
+# Octo variants
+OCTO_VARIANT_STANDARD: Final = "standard"
+OCTO_VARIANT_STAR2: Final = "star2"
+OCTO_VARIANTS: Final = {
+    VARIANT_AUTO: "Auto-detect (recommended)",
```

Ruff (0.14.11)

Undefined name `VARIANT_AUTO`

(F821)